CSC8503CoreClasses/RenderObject.h

```diff
@@ -45,12 +45,17 @@ namespace NCL {
 				return colour;
 			}
 
+			float GetCullSphereRadius() const {
+				return mCullSphereRadius;
+			}
+
 		protected:
 			Mesh*		mesh;
 			Texture*	texture;
 			Shader*		shader;
 			Transform*	transform;
 			Vector4		colour;
+			float		mCullSphereRadius;
 		};
 	}
 }
```

Comment:
Every change looks like no problem.
But I don't know how to pull and run the code that needs to be reviewed.
@Eren, It's up to you.